modules/auxiliary/scanner/http/exchange_proxylogon.rb

```diff
@@ -0,0 +1,84 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Auxiliary
+  include Msf::Exploit::Remote::HttpClient
+  include Msf::Auxiliary::Scanner
+  include Msf::Auxiliary::Report
+
+  def initialize(info = {})
+    super(update_info(info,
+                      'Name' => 'Microsoft Exchange ProxyLogon',
+                      'Description' => '
+        This module scan for a vulnerability on Microsoft Exchange Server that allows an attacker bypassing
+        the authentication and impersonating as the admin (CVE-2021-26855). By chaining this bug with another
+        post-auth arbitrary-file-write vulnerability to get code execution (CVE-2021-27065).
+
+        As a result, an unauthenticated attacker can execute arbitrary commands on Microsoft Exchange Server.
+
+        This vulnerability affects (Exchange 2013 Versions < 15.00.1497.012, Exchange 2016 CU18 < 15.01.2106.013,
+        Exchange 2016 CU19 < 15.01.2176.009, Exchange 2019 CU7 < 15.02.0721.013, Exchange 2019 CU8 < 15.02.0792.010).
+
+        All components are vulnerable by default.
+      ',
+                      'Author' => [
+                        'mekhalleh (RAMELLA Sébastien)' # Module author (Zeop Entreprise)
+                      ],
+                      'References' => [
+                        %w[CVE 2021-26855],
+                        ['URL', 'https://proxylogon.com/'],
+                        ['URL', 'https://raw.githubusercontent.com/microsoft/CSS-Exchange/main/Security/http-vuln-cve2021-26855.nse'],
+                        ['URL', 'http://aka.ms/exchangevulns']
```

Comment:
Using `%w[]` format is acceptable, but should not be mixed and matched.

```suggestion
                        ['CVE', '2021-26855'],
                        ['URL', 'https://proxylogon.com/'],
                        ['URL', 'https://raw.githubusercontent.com/microsoft/CSS-Exchange/main/Security/http-vuln-cve2021-26855.nse'],
                        ['URL', 'http://aka.ms/exchangevulns']
```

Also, optionally:

```ruby
['LOGO', 'https://proxylogon.com/images/logo.jpg'],
```